common/src/main/java/io/druid/common/guava/GuavaUtils.java

```diff
@@ -95,4 +95,24 @@ public InputStreamReader getInput() throws IOException
       }
     };
   }
+
+  // should be used only for estimation.. does not check validity of format
+  public static int binaryLengthAsUTF8(String value)
```

Comment:
any reason we put this in GuavaUtils? It does not really seem to be related to Guava.